integration_tests/src/main/python/qa_nightly_select_test.py

```diff
@@ -134,6 +146,8 @@ def idfn(val):
         'spark.rapids.sql.hasNans': 'false',
         'spark.rapids.sql.castStringToFloat.enabled': 'true',
         'spark.rapids.sql.castFloatToString.enabled': 'true',
+        # some of the first/last tests need a single partition to work reliably when run on a large cluster.
+        'spark.sql.shuffle.partitions': '1'
```

Comment:
would it be better to split the first/last test to use different confs so we can test others with multiple partitions?

Reply:
Sure simple enough to do.